quickfixj-core/src/main/doc/usermanual/usage/acceptor_failover.html

```diff
@@ -9,10 +9,10 @@ <h1>QuickFIX/J User Manual</h1>
 </div>
 		<h2>Simple Failover for Socket-based Acceptors.</h2>
 		<p>
-		When using a MessageStore that supports shared data (FileStore, JdbcStore,
-			SleepycatStore) and that implements the new <code>RefreshableMessageStore</code> interface,
+		When using a MessageStore that supports shared data (FileStore, JdbcStore and SleepycatStore),
 			the Session can be configured to refresh the store information upon
-			logon. You would typically run two acceptor processes using a
+			logon with configuration <span style="color:grey">RefreshMessageStoreAtLogon=Y</span>.
```

Comment:
Actually the configuration for this is `RefreshOnLogon`.
https://github.com/quickfix-j/quickfixj/blob/a8725ad9a849dbfcb4c8f220538daa721029c101/quickfixj-core/src/main/java/quickfix/Session.java#L299

Reply:
Good catch, updated.